client/views/monitorings/herptiles.html

```diff
@@ -140,6 +140,14 @@ <h1 class="page-header">
     </div>
   </div>
 
+  <div class="row">
+    <div class="col-md-12">
+      <field name="threats" type="multiple-choice" nomenclature="main_threats"
+             label="{{'COMMON_FORM_THREATS' | translate}}"
+             model="monitoringDetailController.data.threats"></field>
+    </div>
+  </div>
+
   <div class="row">
     <div class="col-md-12">
       <field name="speciesNotes" type="memo"
```

Comment:
`speciesNote` and `threats` seems to be in every form, maybe extract as partial template?